src/EdFi.AnalyticsMiddleTier.DataStandard31/Asmt/MSSQL/0000-View-AssessmentDim-Create.sql

```diff
@@ -0,0 +1,36 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'asmt_AssessmentDim')
+BEGIN
+	DROP VIEW analytics.asmt_AssessmentDim
+END
+GO
+
+CREATE VIEW analytics.asmt_AssessmentDim
+AS
+	SELECT 
+       Assessment.AssessmentIdentifier AS AssessmentKey, 
+       Assessment.AssessmentTitle AS Title, 
+       COALESCE(Assessment.AssessmentVersion,0) AS Version, 
+       Descriptor.CodeValue AS Category, 
```

Comment:
Based on the ticket description, it should be the description what we return, instead of the CodeValue.